codegen/src/main/twirl/templates/PlayJavaServer/Router.scala.txt

```diff
@@ -25,23 +27,25 @@ import io.grpc.Status;
  * Abstract base class for implementing @{service.name} in Java and using as a play Router
  */
 public abstract class Abstract@{service.name}Router extends PlayRouter implements @{service.name} {
-  private final Function<Throwable, Status> eHandler;
+  private final ActorSystem actorSystem;
```

Comment:
Maybe it would be better to change `PlayRouter` to also requires an `ActorSystem` and not only a `Materializer`. But decide not to change it here to keep the impact of this PR small.

Reply:
I don't get this comment, because it has been changed to also require an actor system now.

Reply:
Sorry, I mean here, at the constructor:

https://github.com/akka/akka-grpc/blob/09de88af1f12a698eda61477e4c2763ed81bf8fb/runtime/src/main/scala/akka/grpc/internal/PlayRouter.scala#L41

Should it instead be:

```scala
@InternalApi abstract class PlayRouter(mat: Materializer, actorSystem: ActorSystem, serviceName: String) extends play.api.routing.Router
```

?

In the end, user extending the generated code will need to inject an `ActorSystem` anyway.